src/mkrelease.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+###############################################################################
+#
+#  ./mkrelease.sh [version]
```

Comment:
why is this bash instead of (I guess) node? (or python :))

Reply:
no better answer then existing dev.sh was in bash, and I didn't want to introduce yet another language. It's quite possible this gets rewritten in node at some point.